examples/load_from_cloud.html

```diff
@@ -0,0 +1,442 @@
+    <br/>
+    <label for="path">Path</label>
+    <input type="text" size=40
+           value="lion_takanawa/cloud.js"
+           name="path" id="path" required>
+    <br/><br/><br/>
+
+    <b>AWS S3</b>
+    <br/>
+    <label for="region">Region</label>
+    <input type="text" size=20
+           placeholder="REGION"
+           name="region" id="aws_region" required>
+    <br/>
+    <label for="bucket">Bucket</label>
+    <input type="text" size=40
+           placeholder="BUCKET"
+           name="bucket" id="aws_bucket" required>
+    <br/>
+    <label for="folder">Folder</label>
+    <input type="text" size=40
+           placeholder="FOLDER"
+           name="folder" id="aws_folder" required>
+    <br/>
+    <label for="uname">AWS Access Key ID</label>
+    <input type="text" size=20
+           placeholder="AWS_ACCESS_KEY_ID"
+           name="uname" id="aws_key" required>
+    <br/>
+    <label for="psw">AWS Secret Access Key</label>
+    <input type="password" size=40
+           placeholder="AWS_SECRET_ACCESS_KEY" name="psw" id="aws_secret" required>
+           <a href="https://aws.amazon.com/blogs/security/wheres-my-secret-access-key/" style="color: gray"> What's this?</a>
+    <br/>
+    <button type="submit" id="aws-start-button">Start</button>
+    </div>
+
+    <br/><br/>
+    <b>Microsoft Azure</b>
+    <br/>
+    <label for="azure_account_name">Account Name</label>
+    <input type="text" size=20
+           placeholder="ACCOUNT_NAME"
+           name="azure_account_name" id="azure_account_name" required>
+    <br/>
+    <label for="container">Container</label>
+    <input type="text" size=20
+           placeholder="CONTAINER"
+           name="container" id="azure_container" required>
+    <br/>
+    <label for="azure_secret_key">Azure Secret Key</label>
+    <input type="password" size=40
+           placeholder="AZURE_SECRET_KEY"
+           name="azure_secret_key" id="azure_secret" required>
+           <a href="https://docs.microsoft.com/en-us/azure/storage/common/storage-account-keys-manage?tabs=azure-portal" style="color: gray">  What's this?</a>
+    <br/>
+    <button type="submit" id="azure-start-button">Start</button>
+    </div>
+
+    <br/><br/><br/>
+    <b>Google Cloud Platform</b>
+    <br/>
+    <label for="gcp_bucket">Bucket</label>
+    <input type="text" size=20
+           placeholder="BUCKET"
+           name="gcp_bucket" id="gcp_bucket" required>
+    <br/>
+
+    <label for="key_file">JSON Key File</label>
+    <input type="file" size=40
+           placeholder="KEY_FILE"
+           name="key_file" id="gcp_key_file" required>
+           <a href="https://cloud.google.com/iam/docs/creating-managing-service-account-keys#iam-service-account-keys-create-console" style="color: gray"> What's this?</a>
+    <br/>
+
+    <button type="submit" id="gcp-start-button">Start</button>
+    </div>
+    <br/>
+    Note: A CORS policy with the following response header must accompany this request (for potree 2.0):
```

Comment:
Not "potree 2.0". Rather, for point cloud files generated by PotreeConverter 2.0.

[There will be a potree 2.0, but that's not here, yet.]